input/fsh/valuesets/ValueSetMaritalStatusVitalRecords.fsh

```diff
@@ -0,0 +1,14 @@
+ValueSet: ValueSetMaritalStatusVitalRecords
+Id: ValueSet-marital-status-vr
+Title: "Marital Status Value Set"
+Description: "The set of codes used to indicate the marital status of the decedent
+
+Mapping to IJE codes [here](ConceptMap-MaritalStatusCM.html)."
+* $v3-MaritalStatus#D "Divorced"
+* $v3-MaritalStatus#L "Legally Separated"
+* $v3-MaritalStatus#M "Married"
+* $v3-MaritalStatus#S "Never Married"
+* $v3-MaritalStatus#W "Widowed"
+* $v3-MaritalStatus#T "Domestic Partnership" // not currently supported by NCHS
```

Comment:
Should it be included in the VS if not supported?

Reply:
Probably more confusing not to....I'll comment it out...

Reply:
I'll defer to your better judgement